src/applications/facility-locator/utils/selectors.js

```diff
@@ -1,4 +1,4 @@
-export const toggleValues = state => state.featureToggles || {};
-export const isProduction = state => toggleValues(state).production;
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
```

Comment:
if you have a selector file you can import the toggle values selector here. You can see how this is being used below. 

We don't really use `reselect` heavily but this is a good use case for it. might refactor the toggleValues selector to use it so it gets memoized.